templates/chart-adr.html

```diff
@@ -7,10 +11,14 @@
 optionAdr = null;
 
 optionAdr = {
-responsive: true,
-maintainAspectRatio: false,
-height: '100%',
-width: '100%',
+    title: {
+        text: 'please update title text if you know what this data represents'
```

Comment:
The title should be something like "Average Damage per Round". You can also leave out the titles for both charts if you combine them and use the legend instead.